packages/ui/app.json

```diff
@@ -31,6 +31,17 @@
     "web": {
       "favicon": "./src/assets-native/favicon.png"
     },
-    "plugins": ["expo-font"]
+    "plugins": [
+      "expo-font",
+      {
+        "fonts": [
+          "../assets-native/fonts/SpaceMono-Regular.ttf",
+          "../assets-native/fonts/ABCDiatype-Regular.otf",
+          "../assets-native/fonts/ABCDiatype-Light.otf",
+          "../assets-native/fonts/ABCDiatype-Medium.otf",
+          "../assets-native/fonts/marche-super-pro.otf"
```

Comment:
Can we rename to be all the same casing?

Reply:
Let me check this.

Reply:
The fonts aren't broken that I can see, do we need them in both places?

Reply:
Nvm, think I am wrong here.